libafl_frida/build.rs

```diff
@@ -1,5 +1,85 @@
 // build.rs
 #![forbid(unexpected_cfgs)]
+#![allow(deprecated)]
+use {
+    anyhow::{anyhow, Result},
+    bindgen::CargoCallbacks,
+    reqwest::blocking::get,
+    std::{env, io::Cursor, path::Path},
+    tar::Archive,
+    xz2::read::XzDecoder,
+};
+
+fn extract() -> Result<()> {
```

Comment:
Yeah. Mostly likely. Seems there is only support for the `gum` devkit right now, we would need to replace it (probably a crate config option) with `gumjs`. Since one is a superset of the other.